### PR TITLE
Improve release notes

### LINKED ISSUE #188: 🚧Improve release notes
### I have done the items below . . .

- [X] I have updated the title by replacing the '**_<title_**>' section.

### Description

Add the 2 production and preview release note template files to the **_~/ReleaseNotes_** directory.
Also update all of the release notes to match the release note templates in regards to emojis and layout.

**Template Files:**
- [Preview-Release-Notes-TEMPLATE.md](https://github.com/KinsonDigital/Velaptor/files/9125795/Preview-Release-Notes-TEMPLATE.md)
- [Production-Release-Notes-TEMPLATE.md](https://github.com/KinsonDigital/Velaptor/files/9125798/Production-Release-Notes-TEMPLATE.md)

Also add links to the github issues for all of the release notes for version **_v1.0.0-preview.8_**.

Make the issue link layout match the release notes for version **_v1.0.0-preview.9_**

### Acceptance Criteria

**This issue is finished when:**
- [x] Update all of the emojis for the existing release notes
- [x] Release note template files added
- [x] Issue links for each item in **_v1.0.0-preview.8_** release notes updated to new style.
- [x] All release notes refactored

### ToDo Items

- [x] Draft pull request created and linked to this issue
- [X] Priority label added to issue (**_low priority_**, **_medium priority_**, or **_high priority_**)
- [x] Issue linked to the proper project
- [X] Issue linked to proper milestone

### Issue Dependencies

_No response_

### Related Work

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by CalvinWilkinson**
<h1 style="font-weight:bold" align="center">Preview Feature Pull Request</h1>
<h2 style="font-weight:bold" align="center">✅Code Review Checklist✅</h2>

- [x] The **_head(source)_** branch for this pull request is a **_preview feature_** branch, with the correct naming syntax, in the following example:
  - A **_head(source)_** branch is the branch being merged into a target branch.
    <details closed><summary>Example</summary>

      ``` xml
      Syntax: preview/feature/<issue-num>-<description>
      Example: preview/feature/123-my-preview-feature
      ```
    </details>

- [x] The **_head(source)_** branch for this pull request is created from a **_preview release_** branch, with the correct naming syntax, in the following example:
  <details closed><summary>Example</summary>

    ``` xml
    Syntax: preview/v<major>.<minor>.<patch>-preview.<prev-num>
    Example: preview/v1.2.3-preview.4
    ```
  </details>

- [x] The **_base(target)_** branch for this pull request is a **_preview release_** branch, with the correct naming syntax, in the following example:
  - A **_base(target)_** branch is the branch that the **_head(source)_** branch is merging into.

    <details closed><summary>Example</summary>

      ``` xml
      Syntax: preview/v<major>.<minor>.<patch>-preview.<prev-num>
      Example: preview/v1.2.3-preview.4
      ```
    </details>

💡For more information on branching, refer to the projects [branching documentation](../../Documentation/Branching.md).

- [x] Pull request title matches the title of the associated issue.

- [x] Associated issue exists and is linked to this pull request.
  - One issue per pull request.

- [x] The labels attached to this PR match the labels attached to the associated issue.

- [x] My code follows the coding style of this project.
  - The style is enforced by the **_.editorconfig_** files in the project and displayed as warnings.  If there is an edge case, reach out and let's discuss it.

- [x] A preview-label label has been added to the pull request.

- [x] I have manually tested my code changes to the best of my ability.

- [x] All tests passed locally.
  - This is required because unit tests might pass locally but not in the CI environment during the status check process or vice-versa.
  - Tests might pass on the developer's machine but not necessarily on the code reviewer's machine.
  - This does not mean that the tests pass in both the local and CI environment.
  - Status checks run for this pull request when merging into the **_preview/v\*.\*.\*-preview.\*_** branch.  These status checks run every time a change is pushed to the pull request.  These checks validate version syntax, tagging, builds, unit tests, and more.

### CODE REVIEW

**Review by KinsonDigitalAdmin — DISMISSED** [review]

File: Documentation/ReleaseNotes/PreviewReleases/Release-Notes-v1.0.0-preview.7.md [hunk 1]
```diff
@@ -6,10 +6,9 @@ As with all software, including releases, there is always a chance for issues an
 
 ---
 
-<h2 style="font-weight:bold" align="center">New 🎉</h2>
+<h2 style="font-weight:bold" align="center">New Features ✨</h2>
 
-1. Add ability to render rectangles to the scene with the various attributes below:  
-   GitHub Issues: [#75](https://github.com/KinsonDigital/Velaptor/issues/75)
+1. [#75](https://github.com/KinsonDigital/Velaptor/issues/75) - Add ability to render rectangles to the scene with the various attributes below:
```
Comment (kselena) [inline Documentation/ReleaseNotes/PreviewReleases/Release-Notes-v1.0.0-preview.7.md:11]: Refactor to match below:

```md
   - Solid/filled rectangles
   - Empty rectangles
   - Change the fill color of a filled rectangle
   - Change the border color of an empty rectangle
   - Add a horizontal color gradient with a start and end color
     - This can be applied to filled and empty rectangles
   - Add a vertical color gradient with a start and end color
     - This can be applied to filled and empty rectangles
   - Change the border thickness of an empty rectangle
   - Set the radius of each corner independently
     - This can be applied to filled and empty rectangles
```

File: Documentation/ReleaseNotes/PreviewReleases/Release-Notes-v1.0.0-preview.9.md [hunk 2]
```diff
@@ -76,6 +77,7 @@ As with all software, including releases, there is always a chance for issues an
 
 ---
 
-<h2 style="font-weight:bold" align="center">Other 👏</h2>
+<h2 style="font-weight:bold" align="center">Other 🪧</h2>
+<h5 align="center">(Includes anything that does not fit in any of the categories above)</h5>
 
 1. [#254](https://github.com/KinsonDigital/Velaptor/issues/254) - Created new issue template for performing releases
```
Comment (kselena) [inline Documentation/ReleaseNotes/PreviewReleases/Release-Notes-v1.0.0-preview.9.md:83]: Add period
```suggestion
1. [#254](https://github.com/KinsonDigital/Velaptor/issues/254) - Created new issue template for performing releases.
```

**Review by kselena — CHANGES_REQUESTED** [review]

File: Documentation/ReleaseNotes/NonReleases/Release-Notes-v0.6.0.md [hunk 3]
```diff
@@ -4,12 +4,12 @@
 
 1. Updated **SDLCore** nuget package from **v0.0.1** to **v0.1.0**
 
-### **Changes** ✨
+### **Changes**
 
 2. Refactored code according to **Microsoft's** FxCop analyzers as well as setting code base to use nullable references
```
[the remark this answers is not in the document]
Comment (CalvinWilkinson) [reply Documentation/ReleaseNotes/NonReleases/Release-Notes-v0.6.0.md:9]: I changed it to:

```suggestion
2. Refactored code according to **Microsoft's** FxCop analyzers as well as setting up code base to use nullable references
```

File: Documentation/ReleaseNotes/PreviewReleases/Release-Notes-v1.0.0-preview.3.md [hunk 4]
```diff
@@ -107,7 +100,10 @@ As with all software, there is always a chance for issues and bugs to exist with
 
 ---
 
-<h2 style="font-weight:bold" align="center">Other 👏</h2>
+<h2 style="font-weight:bold" align="center">Other 🪧</h2>
+<h5 align="center">(Includes anything that does not fit in any of the categories above)</h5>
```
[the remark this answers is not in the document]
Comment (CalvinWilkinson) [reply Documentation/ReleaseNotes/PreviewReleases/Release-Notes-v1.0.0-preview.3.md:104]: I went ahead and updated all the rest of the release notes for this change as well.

**Review by KinsonDigitalAdmin — DISMISSED** [review]

**Review by KinsonDigitalAdmin — DISMISSED** [review]

**Review by kselena — APPROVED** [review]

**Review by KinsonDigitalAdmin — APPROVED** [review]
